lib/components/primitive-components/Trace/Trace.ts

```diff
@@ -777,6 +793,29 @@ export class Trace
           connectedTo: [],
         })
       }
+      if (
+        elm.type === "schematic_net_label" &&
+        (elm.text.toLowerCase().includes("gnd") ||
+          elm.text.toLowerCase().includes("ground"))
```

Comment:
We should adapt this code to derive the symbol width dynamically, but we can do this in a future issue, I'm leaving this comment so that you can use "..." -> "Create Issue" after this is merged!

```suggestion
        elm.type === "schematic_net_label" &&
        elm.symbol_name
```